test/resources/scan_form.test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable func-names */
+import { expect } from 'chai';
+import * as setupPolly from '../setup_polly';
+import EasyPost from '../../src/easypost';
+import Fixture from '../helpers/fixture';
+import NotImplementedError from '../../src/errors/not_implemented';
+
+describe('ScanForm Resource', function () {
+  setupPolly.startPolly();
+
+  before(function () {
+    this.easypost = new EasyPost(process.env.EASYPOST_TEST_API_KEY);
+  });
+
+  beforeEach(function () {
+    const { server } = this.polly;
+    setupPolly.stripCreds(server);
+  });
+
+  it('creates a scanform', async function () {
+    const shipment = await new this.easypost.Shipment(Fixture.oneCallBuyShipment()).save();
+
+    const scanform = await new this.easypost.ScanForm({
+      shipments: [shipment],
+    }).save();
+
+    expect(scanform).to.be.an.instanceOf(this.easypost.ScanForm);
+    expect(scanform.id).to.match(/^sf_/);
+  });
+
+  it('retrieves a scanform', async function () {
+    const shipment = await new this.easypost.Shipment(Fixture.oneCallBuyShipment()).save();
+
+    const scanform = await new this.easypost.ScanForm({
+      shipments: [shipment],
+    }).save();
+
+    const retrievedScanform = await this.easypost.ScanForm.retrieve(scanform.id);
+
+    expect(retrievedScanform).to.be.an.instanceOf(this.easypost.ScanForm);
+    delete scanform.shipments; // This lib returns the shipments key as part of the response, remove it for comparison
+    expect(retrievedScanform.id).to.equal(scanform.id);
```

Comment:
Can we compare the objects here?

Reply:
Ah yes, this was leftover from some bugs I had to deal with. Fixed